maintainers/maintainer-list.nix

```diff
@@ -13953,4 +13953,16 @@
     github = "nigelgbanks";
     githubId = 487373;
   };
+  zanculmarktum = {
```

Comment:
Each maintainer must be added in it's own seperate commit

The commit message must be as follow `maintainers: add <name>`

Reply:
Updated commits. Sorry for the back and forth.